File: C++/Iniciante/1038.cpp
#include <stdio.h>
#include <iostream>

int main() {

  int cod, qtd;
  double valor;
  std::cin >> cod >> qtd;

  switch(cod){
    case 1:
      valor = qtd * 4.00;
      break;
    
    case 2:
      valor = qtd * 4.50;
      break;
    
    case 3:
      valor = qtd * 5.00;
      break;
    
    case 4:
      valor = qtd * 2.00;
      break;
    
    case 5:
      valor = qtd * 1.50;
      break;
  }

  printf("Total: R$ %.2lf\n", valor);

}

File: Python/Iniciante/1038.py
cod, qtd = input().split()
cod = int(cod)
qtd = int(qtd)

itens = {
  1: 4.00,
  2: 4.50,
  3: 5.00,
  4: 2.00,
  5: 1.50
}

item = itens[cod]
valor = item * qtd

print(f"Total: R$ {valor:.2f}")
